azure-pipelines-data-container.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  - master
+  - datacontainerpipelinereview
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Nightly build
+  branches:
+    include:
+    - master
+  always: true
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+variables:
+  IMAGE_TAG_PREFIX: $[format('1.0.{0:HHm}', pipeline.startTime)]
```

Comment:
Didn't get the idea of this TAG_PREFIX... What do you want it to look like?

Reply:
explained in PR description.
_The image name will be of the format 'major.minor.datetime.build' (ex: 1.0.21320200622.6)_